conversions/weight_conversion.py

```diff
@@ -0,0 +1,287 @@
+    1360776.0
+    >>> weight_conversion("pound","metric-ton",3)
+    0.001360776
+    >>> weight_conversion("pound","long-ton",2)
+    0.0008928571152432
+    >>> weight_conversion("pound","short-ton",1)
+    0.0004999999954224
+    >>> weight_conversion("pound","pound",3)
+    3.0000000000220752
+    >>> weight_conversion("pound","ounce",1)
+    16.000000000027015
+    >>> weight_conversion("pound","carrat",1)
+    2267.96
+    >>> weight_conversion("pound","atomic-mass-unit",4)
+    1.0926372033015936e+27
+    >>> weight_conversion("ounce","kilogram",3)
+    0.0850485
+    >>> weight_conversion("ounce","gram",3)
+    85.0485
+    >>> weight_conversion("ounce","milligram",4)
+    113398.0
+    >>> weight_conversion("ounce","metric-ton",4)
+    0.000113398
+    >>> weight_conversion("ounce","long-ton",4)
+    0.0001116071394054
+    >>> weight_conversion("ounce","short-ton",4)
+    0.0001249999988556
+    >>> weight_conversion("ounce","pound",1)
+    0.0625000000004599
+    >>> weight_conversion("ounce","ounce",2)
+    2.000000000003377
+    >>> weight_conversion("ounce","carrat",1)
+    141.7475
+    >>> weight_conversion("ounce","atomic-mass-unit",1)
+    1.70724563015874e+25
+    >>> weight_conversion("carrat","kilogram",1)
+    0.0002
+    >>> weight_conversion("carrat","gram",4)
+    0.8
+    >>> weight_conversion("carrat","milligram",2)
+    400.0
+    >>> weight_conversion("carrat","metric-ton",2)
+    4.0000000000000003e-07
+    >>> weight_conversion("carrat","long-ton",3)
+    5.9052438e-07
+    >>> weight_conversion("carrat","short-ton",4)
+    8.818497600000002e-07
+    >>> weight_conversion("carrat","pound",1)
+    0.00044092488404000004
+    >>> weight_conversion("carrat","ounce",2)
+    0.0141095962892
+    >>> weight_conversion("carrat","carrat",4)
+    4.0
+    >>> weight_conversion("carrat","atomic-mass-unit",4)
+    4.8177093216e+23
+    >>> weight_conversion("atomic-mass-unit","kilogram",4)
+    6.642160796e-27
+    >>> weight_conversion("atomic-mass-unit","gram",2)
+    3.321080398e-24
+    >>> weight_conversion("atomic-mass-unit","milligram",2)
+    3.3210803980000002e-21
+    >>> weight_conversion("atomic-mass-unit","metric-ton",3)
+    4.9816205970000004e-30
+    >>> weight_conversion("atomic-mass-unit","long-ton",3)
+    4.9029473573977584e-30
+    >>> weight_conversion("atomic-mass-unit","short-ton",1)
+    1.830433719948128e-30
+    >>> weight_conversion("atomic-mass-unit","pound",3)
+    1.0982602420317504e-26
+    >>> weight_conversion("atomic-mass-unit","ounce",2)
+    1.1714775914938915e-25
+    >>> weight_conversion("atomic-mass-unit","carrat",2)
+    1.660540199e-23
+    >>> weight_conversion("atomic-mass-unit","atomic-mass-unit",2)
+    1.999999998903455
+    """
+    try:
+        return value * KILOGRAM_CHART[to_type] * WEIGHT_TYPE_CHART[from_type]
+    except TypeError:
+        return "Value type is not float"
+    except Exception as e:
+        return str(e) + " is not known weight unit"
```

Comment:
```suggestion
    except KeyError as err:
        print(
            f"Invalid 'from_type' or 'to_type' value: {err}\n"
            f"Supported values are: {', '.join(WEIGHT_TYPE_CHART)}"
        )
```
Multiplying an `int` to a `float` does not raise any exception. As you're accessing a key in a dictionary, if the key is not present it will raise a `KeyError`.